src/EnergyPlus/DataDaylighting.hh

```diff
@@ -196,9 +199,9 @@ namespace DataDaylighting {
         //  3: Reference point number (1 to Total Daylighting Reference Points)
         //  4: Sky type (1 to 4; 1 = clear, 2 = clear turbid, 3 = intermediate, 4 = overcast
         //  5: Daylit window number (1 to NumOfDayltgExtWins)
-        Array5D<Real64> DaylIllFacSky;
-        Array5D<Real64> DaylSourceFacSky;
-        Array5D<Real64> DaylBackFacSky;
+        Array4D<Dayltg::Illums> DaylIllFacSky;
```

Comment:
I reorganized the dimensions and now I see that I need to update the comment also. The fifth dimension is `SkyType`, and by factoring that out into a `std::array`, the remaining dimensions can be contained in an `Array4D`.